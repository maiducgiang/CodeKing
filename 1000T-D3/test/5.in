5
8241 3133 2354 8536
4123 6458 5310 8013
8858 2575 8831 5314
9799 8242 262 3812
2567 5696 2551 2626
