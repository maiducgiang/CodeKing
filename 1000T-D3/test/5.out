26774396
45436449
69168723
4726742
14723021
